.ci/helper_shellchecks.sh

```diff
@@ -5,7 +5,7 @@ CI_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source "${CI_DIR}/common.sh"
 
 # shellcheck disable=2016
-mapfile -t shellscript_locations < <({ ggit grep -lE '^#!(/usr)?/bin/(env )?(bash|sh)' -- ':(exclude)plugins/terminal.koplugin/shfm' && git submodule --quiet foreach '[ "$path" = "base" -o "$path" = "platform/android/luajit-launcher" ] || git grep -lE "^#!(/usr)?/bin/(env )?(bash|sh)" | sed "s|^|$path/|"' && git ls-files ./*.sh; } | sort | uniq)
+mapfile -t shellscript_locations < <({ git grep -lE '^#!(/usr)?/bin/(env )?(bash|sh)' -- ':(exclude)plugins/terminal.koplugin/shfm' && git submodule --quiet foreach '[ "$path" = "base" -o "$path" = "platform/android/luajit-launcher" ] || git grep -lE "^#!(/usr)?/bin/(env )?(bash|sh)" | sed "s|^|$path/|"' && git ls-files ./*.sh; } | sort | uniq)
```

Comment:
Doesn't seem to work either :?

Reply:
Oh, it works just fine, just apparently not without https://github.com/koreader/koreader/pull/7775

In that case could you make it `git grep -lE '^#!(/usr)?/bin/(env )?(bash|sh)' | sed "/^plugins\/terminal.koplugin\/shfm$/d"`? It's primarily the location where you put it I find very illegible, the method itself merely a bit inelegant.